app/qml/components/ProjectList.qml

```diff
@@ -0,0 +1,355 @@
+
+      projectSourceModel: ProjectsModel {
+        id: controllerModel
+
+        merginApi: __merginApi
+        localProjectsManager: __localProjectsManager
+        modelType: root.projectModelType
+      }
+    }
+
+    // Project delegate
+    delegate: ProjectDelegateItem {
+      id: projectDelegate
+
+      width: parent.width
+      height: InputStyle.rowHeightHeader * 1.2
+
+      projectDisplayName: root.projectModelType === ProjectsModel.CreatedProjectsModel ? model.ProjectName : model.ProjectFullName
+      projectId: model.ProjectId
+      projectDescription: model.ProjectDescription
+      projectStatus: model.ProjectSyncStatus ? model.ProjectSyncStatus : ProjectStatus.NoVersion
+      projectIsValid: model.ProjectIsValid
+      projectIsPending: model.ProjectPending ? model.ProjectPending : false
+      projectSyncProgress: model.ProjectSyncProgress ? model.ProjectSyncProgress : -1
+      projectIsLocal: model.ProjectIsLocal
+      projectIsMergin: model.ProjectIsMergin
+      projectRemoteError: model.ProjectRemoteError ? model.ProjectRemoteError : ""
+
+      highlight: model.ProjectId === root.activeProjectId
+
+      viewContentY: ListView.view.contentY
+      viewHeight: ListView.view.height
+
+      onOpenRequested: {
+        if ( model.ProjectIsLocal )
+          root.openProjectRequested( projectId, model.ProjectFilePath )
+        else if ( !model.ProjectIsLocal && model.ProjectIsMergin ) {
+          downloadProjectDialog.relatedProjectId = model.ProjectId
+          downloadProjectDialog.open()
+        }
+      }
+      onSyncRequested: {
+        if ( __inputUtils.hasStoragePermission() ) {
+          controllerModel.syncProject( projectId )
+        }
+        else if ( __inputUtils.acquireStoragePermission() ) {
+          restartAppDialog.open()
+        }
+      }
+      onMigrateRequested: controllerModel.migrateProject( projectId )
+      onRemoveRequested: {
+        removeDialog.relatedProjectId = projectId
+        removeDialog.open()
+      }
+      onStopSyncRequested: controllerModel.stopProjectSync( projectId )
+      onShowChangesRequested: root.showLocalChangesRequested( projectId )
+    }
+  }
+
+  Component {
+    id: fetchMoreButtonComponent
+
+    DelegateButton {
+      width: parent.width
+      height: visible ? InputStyle.rowHeight : 0
+      text: qsTr( "Fetch more" )
+
+      visible: controllerModel.hasMoreProjects
+      onClicked: controllerModel.fetchAnotherPage( viewModel.searchExpression )
+    }
+  }
+
+  Component {
+    id: addProjectButtonComponent
+
+    DelegateButton {
+      width: parent.width
+      height: InputStyle.rowHeight
+      text: qsTr("Create project")
+      visible: listview.count > 0
+
+      onClicked: {
+        if ( __inputUtils.hasStoragePermission() ) {
+          stackView.push(projectWizardComp)
+        }
+        else if ( __inputUtils.acquireStoragePermission() ) {
+          restartAppDialog.open()
+        }
+      }
+    }
+  }
+
+  Text {
```

Comment:
cannot we use RichTextBlock  comoibbebt

Reply:
Google knows us well 😄  ![image](https://user-images.githubusercontent.com/22449698/114378303-dddbec80-9b87-11eb-9f80-4cde570e70eb.png)